### PR TITLE
Add Vespa icon

### LINKED ISSUE #9290: Add Vespa icon
### Brand Name

Vespa

### Website

https://www.vespa.com

### Popularity Metric

Similarweb Rank: 57,046. See https://www.similarweb.com/website/vespa.com

### Official Resources for Icon and Color

SVG: Taken from [logo guide PDF](https://www.piaggiogroup.com/sites/piaggiocorpd7/files/doc/guidalogo_vespa_0.pdf)
Color: `#000000` As mentioned in the logo guide

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
vespa

**Issue:** closes #9290 

**Similarweb rank:** 57,046. See https://www.similarweb.com/website/vespa.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from [logo guide PDF](https://www.piaggiogroup.com/sites/piaggiocorpd7/files/doc/guidalogo_vespa_0.pdf)
Color: `#000000` As mentioned in the logo guide

### CODE REVIEW

**Review by FraneCaleta — APPROVED** [review]

File: icons/vespa.svg
[the remark this answers is not in the document]
Comment (LitoMore) [reply icons/vespa.svg:1]: We should. See their products at https://www.vespa.com. The brand mark on the motocycle does not have the `®` sign.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13395,6 +13395,12 @@
             "hex": "CD040B",
             "source": "https://www.verizondigitalmedia.com/about/logo-usage/"
         },
+        {
+            "title": "Vespa",
+            "hex": "000000",
```
[the remark this answers is not in the document]
Comment (PratyushJayachandran) [reply _data/simple-icons.json:13400]: According to the logo guide I attached, if you want colors, you need to use the version with black outline. Is my interpretation correct?

File: icons/vespa.svg
[the remark this answers is not in the document]
Comment (PratyushJayachandran) [reply icons/vespa.svg:1]: I have removed the ® sign.

File: _data/simple-icons.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamrusted) [reply _data/simple-icons.json:13400]: I would say we can get away with this one. The icon will still be noticeably Vespa regardless of whether or not we include the outline.

**Comment by adamrusted** [comment]
Icon is spot on @PratyushJayachandran - just the change to the hex code and we'll get this one merged in!

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PratyushJayachandran 🎉

**Pull request merged**